tests/Fixtures/Integration/priority/php_unit_data_provider_name,php_unit_attributes.test

```diff
@@ -1,49 +0,0 @@
---TEST--
```

Comment:
let's never remove integration test on priority, only move to misc when no longer needed on priority checks.

Reply:
recovered